frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -13,9 +13,11 @@ async function parsePrismaSchema(schemaString: string): Promise<ProcessResult> {
   const relationships: Record<string, Relationship> = {}
   const errors: Error[] = []
 
+  const modelNames = dmmf.datamodel.models.map((model) => model.name)
   for (const model of dmmf.datamodel.models) {
     const columns: Columns = {}
     for (const field of model.fields) {
+      if (modelNames.includes(field.type)) continue
```

Comment:
The following code might work 👀
Please check it out!

```suggestion
  for (const model of dmmf.datamodel.models) {
    const columns: Columns = {}
    for (const field of model.fields) {
      if (field.relationName) continue
```

Reply:
https://github.com/liam-hq/liam/pull/533/commits/519b09aeeae87b5e2b98dca3aa37dd2bc259d73d I've checked it works. No difference in schema.json